Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml

```
@@ -56,6 +59,9 @@
   - type: ExplosionResistance
     damageCoefficient: 0.9
   - type: GroupExamine
+  - type: TemperatureProtection # Mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)